velox/expression/tests/CastExprTest.cpp

```diff
@@ -1346,6 +1358,10 @@ TEST_F(CastExprTest, decimalToFloat) {
   testDecimalToFloatCasts<double>();
 }
 
+TEST_F(CastExprTest, decimalToBool) {
+  testDecimalToBoolCasts();
```

Comment:
nit: theres no need to create a separate function here right (previous cases are templates) ? Lets just move the tests in here ?

Reply:
Thanks, updated it.